docs/whitepaper.tex

```diff
@@ -247,38 +247,98 @@ \section{Tests}
 
 \todo{Should we write Apropos-Tx tests?}
 
-\subsection{Unit tests}
+\subsection{Components}
+
+Tests are broken down by the onchain component they test.
+
+\subsubsection{Config utxo tests}
+
+NFT tests
 
-The following intended behaviors need to work:
 \begin{itemize}
-  \item Initializing a protocol and pool works
-  \item queries accurately reflect how the pool was setup
-  \item valid swaps work
-  \item the queries correctly report which swaps would be valid
-  \item queries accurately reflect the change from a swap
-  \item valid liquidity changes work
-  \item queries reflect the liquidity changes
-  \item admin can kill a pool
-  \item liquidity can be extracted from a dead pool
+	\item Minting an NFT actually produces an NFT
+	\item Minting an NFT fails if the seed Tx is a reference input
+	\item Trying to mint the NFT twice fails
+	\item The seed Tx has been spent after minting an NFT
+	\item The NFT can't be burned
```

Comment:
What do these tests look like, exactly? Some of these are properties, not tests.